docs/template_README.md

```diff
@@ -0,0 +1,22 @@
+# repo_name
+
+[![License](https://img.shields.io/:license-apache-blue.svg)](http://www.apache.org/licenses/LICENSE-2.0.html)
+
+## What is repo_name?
+
+The repo_name is ...
+
+Go to the [Contributing guide](CONTRIBUTING.md) to learn how to get involved.
+
+## Getting started
+
+- Steps for development:
+
+- Steps for deployment:
+
+- Check the [Security guide](SECURITY.md) if you need to report a security issue.
+
+## References
+
+- repo_name is part of the open-cluster-management community. For more information, visit: [open-cluster-management.io](https://open-cluster-management.io).
+- optional: list of additional references if needed.
```

Comment:
Line 21 might be better in the description of the said repo?

Reply:
I put it in the reference section here so its easier to copy and paste and also makes it stand out a bit more instead of mix it in with the repo description.